test/components/Chat.test.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import ChatView from '../../src/components/Chat';
```

Comment:
"Component" não seria mais descritivo? Ou estamos seguindo algum padrão aqui?

```suggestion
import ChatComponent from '../../src/components/Chat';
```